packages/ag-charts-community/src/util/type-guards.ts

```diff
@@ -49,7 +49,7 @@ export function isNumber(value: unknown): value is number {
 }
 
 export function isFiniteNumber(value: unknown): value is number {
-    return isNumber(value) && Number.isFinite(value);
+    return Number.isFinite(value);
```

Comment:
Unrelated but it saves an additional check (`Number.isFinite` does not cast to a number, unlike `window.isFinite`)